chore!: Bump stackable-operator to 0.92.0 (#299)
stackabletech/listener-operator: CHANGELOG.md

## [Unreleased]

### Added

- Adds new telemetry CLI arguments and environment variables ([#299]).
  - Use `--file-log-max-files` (or `FILE_LOG_MAX_FILES`) to limit the number of log files kept.

>   - The file log rotation period is now configurable via `FILE_LOG_ROTATION_PERIOD`.

Author: ```suggestion
  - Use `--file-log-rotation-period` (or `FILE_LOG_ROTATION_PERIOD`) to configure the frequency of rotation.
```
